portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase

```diff
@@ -36,10 +36,9 @@ definition {
 		}
 	}
 
-	@description = "LPS-166479. Verify that new SVG spritemap is applied on the page"
+	@description = "Verify that new SVG spritemap is applied on the page"
 	@priority = 3
```

Comment:
@CarlosBrichete usually you keep the LPS number in the test description right? if that's the case, then this one should be kept

Reply:
Yes, we do. It is important to have the LPS in order to have the tests tracked with their parent story.

Reply:
ok then @chestofwonders pls re-add this LPS number, I think after this change we're ready to go

Reply:
it's done @dsanz @CarlosBrichete, if we are ok, we can go with it